docs/HARDWARE-VERIFICATION.md

```diff
@@ -0,0 +1,71 @@
+# Verifying EVE on edge nodes
+
+This is a specialized version of EVE-OS installer, that installs EVE-OS on the
+underlying edge node and verifies its compatibility. It checks if the necessary
+drivers are present, tests a static and dynamic networking configurations,
+tests the available storage devices, and tests if EVE-OS can spawn a guest
+successfully. Verification currently supports verification on x86 edge nodes.
+
+## Running the verification image from boot media
+
+Similar to EVE-OS installer image (see [deployment](DEPLOYMENT.md)), we:
+
+1. Produce a disk-based installer image (e.g., by running make verification-raw)
```

Comment:
What is `make verification-raw` doing - building the docker image `lfedge/verification` or producing the raw image file from an existing docker image?